.github/pull_request_template.md

```diff
@@ -7,12 +8,20 @@
 
 ## Types of changes
 
-<!-- What types of changes does your code introduce? Uncomment all that apply: -->
+<!-- Uncomment the main reason for the change (for example, all feat PRs should include docs and test, but only uncomment feat): -->
+
+<!-- feat (a new feature) -->
+<!-- fix (a bug fix) -->
+
```

Comment:
I was thinking that `feat` and `fix` are the most common, so maybe it's convenient to have them at the top? Or should I just add them to the alphabetized list below?

Reply:
Seems more than reasonable to have those at the top